fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -310,9 +313,9 @@
     private static final Function<Triple, ConstraintViolationException> validateMemberRelation = uncheck(x -> {
         final org.apache.jena.graph.Node object = x.getObject();
         if (object.isURI() && x.getPredicate().getURI().equals(HAS_MEMBER_RELATION.toString()) &&
-            object.getURI().equals(CONTAINS.toString())) {
+            isManagedPredicate.test(createProperty(object.getURI()))) {
             return new ServerManagedPropertyException(
-                "The " + HAS_MEMBER_RELATION + " predicate may not take the ldp:contains type. (" +
+                "The " + HAS_MEMBER_RELATION + " predicate may not take the server managed type. (" +
                 CONTAINS + ").");
```

Comment:
Need to update the exception here, it is still outputting CONTAINS. I believe it should be `object.getURI()`